www/includes/easyparliament/templates/html/alert/update-mp.php

```diff
@@ -0,0 +1,72 @@
+<div class="full-page alerts-header alerts-header--jumbo">
+    <div class="full-page__row">
+        <div class="full-page__unit">
+
+            <h1>Update your MP</h1>
+
+          <?php if (isset($data['already_signed_up'])): ?>
+            <div class="alerts-message alerts-message--reminder">
+                <h2>You are already signed up</h2>
+                <p>You are already receiving alerts when your new MP, <?= $data['mp_name'] ?>, speaks in Parliament or receives an answer to a written question.</p>
+                <p><a href="/alert/" class="button radius">Show my email settings</a></p>
+            </div>
+          <?php elseif (isset($data['update'])): ?>
+
+            <div class="alerts-message alerts-message--confirmation-sent">
+                <h2>Confirm your update!</h2>
+                <p>Please confirm that you want to update your alerts from <?= $data['old_mp'] ?> to <?= $data['new_mp'] ?></p>
+                <form class="alerts-form" action="/alert/update-mp/" method="post">
+                <input type="hidden" name="update-alert" value="1">
+                <input type="hidden" name="confirmation" value="<?= $data['confirmation'] ?>">
+                <p><input type="submit" class="button radius" value="Update"></p>
+                </form>
+            </div>
+
+          <?php elseif (isset($data['signedup_no_confirm']) || isset($data['confirmation_received'])): ?>
+
+            <div class="alerts-message alerts-message--confirmation-received">
+                <h2>Thanks for subscribing!</h2>
+                <p>You will now receive alerts when your MP speaks in Parliament or receives an answer to a written question.</p>
+                <?php if (isset($data['user_signed_in'])): ?>
+                <p><a href="/alert/" class="button radius">Show my email settings</a></p>
+                <?php endif ?>
+            </div>
+
+          <?php elseif (isset($data['error'])): ?>
+            <div class="alerts-message alerts-message--error">
+                <h2>Something went wrong</h2>
+                <p>Sorry, we were unable to create this alert. Please <a href="mailto:<?= str_replace('@', '&#64;', CONTACTEMAIL) ?>">let us know</a>. Thanks.</p>
+            </div>
+
+          <?php else: ?>
```

Comment:
Do we want this else condition, should it just redirect to the alert/by-postcode page?